snow/engine/common/bootstrapper.go

```diff
@@ -221,3 +307,32 @@ func (b *Bootstrapper) Disconnected(validatorID ids.ShortID) error {
 	}
 	return nil
 }
+
+func (b *Bootstrapper) RestartBootstrap() error {
+	if b.bootstrapAttempts >= 50 {
```

Comment:
This should be a CLI parameter, and RetryBootstrap should only be used for the repeated executions.

Reply:
Added! 🚀